From: Lennart Poettering <lennart@poettering.net>
Date: Fri, 14 Feb 2014 16:41:03 +0100
Subject: nspawn: if we don't find bash, try sh

(cherry picked from commit 262d10e6bd8e4a0a6a0967fbb5f89a5b989779ac)

Conflicts:
	src/nspawn/nspawn.c
---
 src/nspawn/nspawn.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/nspawn/nspawn.c b/src/nspawn/nspawn.c
index cafc3062..410b8794 100644
--- a/src/nspawn/nspawn.c
+++ b/src/nspawn/nspawn.c
@@ -1499,6 +1499,7 @@ int main(int argc, char *argv[]) {
                         else {
                                 chdir(home ? home : "/root");
                                 execle("/bin/bash", "-bash", NULL, (char**) envp);
+                                execle("/bin/sh", "-sh", NULL, (char**) envp);
                         }
 
                         log_error("execv() failed: %m");
